java-checks/src/test/files/checks/security/LDAPAuthenticatedConnectionCheck.java

```diff
@@ -0,0 +1,47 @@
+import java.util.Hashtable;
+import javax.naming.Context;
+
+class S4433 {
+
+  void method1() {
+    Hashtable<String, String> env = new Hashtable<>();
+    env.put(javax.naming.Context.SECURITY_AUTHENTICATION, "none"); // Noncompliant {{Change authentication to "simple" or stronger.}}
+  }
+
+  void method2() {
+    Hashtable<String, String> env = new Hashtable<>();
+    env.put(Context.SECURITY_AUTHENTICATION, "none"); // Noncompliant {{Change authentication to "simple" or stronger.}}
```

Comment:
When the message is a constant: {{Change authentication to "simple" or stronger.}}
You just need to test it once in this file.